deployment/hasura/metadata/actions.graphql

```diff
@@ -163,10 +163,15 @@ type MerlinSimulationResponse {
 }
 
 type ValidationResponse {
-  errors: [String!]
+  errors: [ValidationNotice!]!
   success: Boolean!
 }
 
+type ValidationNotice {
+  names: [String!]!
```

Comment:
I'd recommend calling it `subjects` here, as well, since "name" could be construed as the name of the validation or notice itself. (Or, to take a different tack -- "name" *of what?*) Whereas "subjects" conveys that the elements are, in fact, the subject of the validation notice.

Reply:
That's fair! Just pushed this